terraform/outputs.tf

```diff
@@ -6,21 +6,18 @@ output "app_name" {
   value       = juju_application.opensearch-dashboards.name
 }
 
-# Required integration endpoints
-
-output "certificates_endpoint" {
-  description = "Name of the endpoint used to integrate with the TLS certificates provider."
-  value       = "certificates"
-}
-
-output "opensearch_client_endpoint" {
-  description = "Name of the endpoint opensearch-client endpoint."
-  value       = "opensearch-client"
+# integration endpoints
+output "requires" {
+  description = "Map of all \"requires\" endpoints"
+  value = {
+    opensearch_client = "opensearch-client"
+    certificates      = "certificates"
```

Comment:
There is a risk here someone creates a 2nd relation for certificates & sets `var.tls = true` at the same time. This can happen b/c "requires.certificates" is just a string that anyone else can consume.

I'd suggest we have it in `provides` instead, then we can "consume" the self-signed-certificates endpoint instead of providing an endpoint to connect with OSD.

Reply:
According to the [TF Spec](https://docs.google.com/document/d/1EG71A2pJ244PQRaGVzGj7Mx2B_bzE4U_OSqx4eeVI1E/edit?tab=t.0) this output variable should in-fact reflect the provides/requires endpoints as defined in the `metadata.yaml`